cmd/create.go

```diff
@@ -34,6 +35,7 @@ func init() {
 	createCmd.Flags().String("secret-key-file", "", "Read this file for the access token for your cloud (Scaleway, EC2)")
 	createCmd.Flags().String("organisation-id", "", "Organisation ID (Scaleway)")
 	createCmd.Flags().String("project-id", "", "Project ID (Packet.com, Google Compute Engine)")
+	createCmd.Flags().String("subscription-id", "", "Subscription ID (Azure)")
```

Comment:
I wonder if we should use project ID?

Reply:
Azure subscription ID is similar to GCE's project ID. However the term `project ID` is unclear to the Azure user what it is. We would need to explain it somewhere if we use `project ID`. So I prefer to use `subscription ID`.

Reply:
I'm still in two minds about this, I think `project-id` might be easier to manage, but you have a good point too.